packages/common/src/filters/selectFilter.ts

```diff
@@ -229,8 +229,14 @@ export class SelectFilter implements Filter {
       values = Array.isArray(values) ? values : [values];
       this.$filterElm.multipleSelect('setSelects', values);
     }
-    this.getValues().length > 0 ? this.$filterElm.addClass('filled') : this.$filterElm.removeClass('filled');
```

Comment:
@mcallegario I took a look at this and we're missing unit tests and I also see duplicate code for the same `filled` styling css class, so would it be possible to ask you to modify the code in 3 areas? If possible, that'd be great and the change could do under your name as contribution. 

The step 1 would be to delete line 232 to 239
```diff
-    if (this.getValues().length > 0) {
-      this.isFilled = true;
-      this.$filterElm?.addClass('filled').siblings('div .search-filter').addClass('filled');
-    } else {
-      this.isFilled = false;
-      this.$filterElm.removeClass('filled');
-      this.$filterElm.siblings('div .search-filter').removeClass('filled');
-    }
```
and replace with this, I'll show the other 2 steps on the next line number
```ts
this.updateFilterStyle(this.getValues().length > 0);
```